contracts/evoting/types/election.go

```diff
@@ -409,3 +416,100 @@ type PubsharesUnits struct {
 	// PubsharesUnit
 	Indexes []int
 }
+
+// AddVoter add a new admin to the system.
+func (form *Form) AddVoter(userID string) error {
```

Comment:
AddOwner and AddVoter are almost the same code, maybe you can put the common code in a separate function and call that?

Reply:
I've created a SciperToInt method